spec/unit/resources/ethernet_network/create_spec.rb

```diff
@@ -27,3 +27,14 @@
     expect(real_chef_run).to create_oneview_ethernet_network('EthernetNetwork1')
   end
 end
+
```

Comment:
Maybe we would not want to implement tests of things that are already being tested on API200